src/main/java/com/beerjournal/breweriana/user/UserController.java

```diff
@@ -22,8 +21,11 @@
     }
 
     @GetMapping
-    ResponseEntity<Collection<UserDto>> getAllUsers() {
-        return new ResponseEntity<>(userService.getAllUsers(), HttpStatus.OK);
+    ResponseEntity<Page<UserDto>> getUsers(
```

Comment:
I am not sure if we should create another get method for users collection, override current one is ok?